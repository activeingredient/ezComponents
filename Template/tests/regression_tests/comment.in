{* Doc comment *}
Hello world.

{*Another docComment *}
blablabla

{ 4 // + 4 
}
{ 2 /* + 2 */ }
