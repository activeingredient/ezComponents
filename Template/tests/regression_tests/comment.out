Hello world.

blablabla

4
2
